library/lib.sh

```diff
@@ -403,7 +403,7 @@ lsrRunPlaybook() {
     if [ "${GET_PYTHON_MODULES:-}" = true ]; then
         ANSIBLE_ENVS[ANSIBLE_DEBUG]=true
     fi
-    cmd="$(lsrArrtoStr ANSIBLE_ENVS) ansible-playbook -i $inventory $skip_tags $limit $test_playbook -vv"
+    cmd="$(lsrArrtoStr ANSIBLE_ENVS) ansible-playbook -i $inventory $skip_tags $limit $test_playbook ${LSR_ANSIBLE_VERBOSITY:--vv}"
```

Comment:
How about assigning a new input argument for this function `local verbosity=$6`?
I think it would be cleaner that way and that's what I usually do.
If keeping LSR_ANSIBLE_VERBOSITY, let's document it in a comment at the top of https://github.com/linux-system-roles/tft-tests/blob/main/tests/general/test.sh